ams/server/src/main/java/com/netease/arctic/server/dashboard/response/IcebergRestErrorCode.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.server.dashboard.response;
+
+
+/**
+ * iceberg rest-catalog api error response code
+ */
+public enum IcebergRestErrorCode {
```

Comment:
Seems unused class.

Reply:
Used in `handleException` in this class

Reply:
should we use code provided by javalin??

Reply:
Those code are defined by iceberg rest api and correlation with iceberg exceptions. so i redefined the http code here.

Reply:
Remove the useless code.